spearhead-blocks/templates/single.html

```diff
@@ -3,11 +3,11 @@
 <!-- wp:group {"layout":{"inherit":true}} -->
 <div class="wp-block-group"><!-- wp:post-date {"textColor":"secondary","fontFamily":"ibm-plex-mono"} /-->
 
-<!-- wp:post-title {"style":{"typography":{"fontSize":"3rem","lineHeight":1.3999999999999999}}} /--></div>
+<!-- wp:post-title {"style":{"typography":{"fontSize":"3rem","lineHeight":1.2}}} /--></div>
 <!-- /wp:group -->
 
-<!-- wp:group {"tagName":"main","lock":{"move":false,"remove":true}} -->
-<main class="wp-block-group"><!-- wp:post-featured-image {"align":"full"} /-->
+<!-- wp:group {"layout":{"type":"constrained"},"style":{"spacing":{"margin":{"top":"var:preset|spacing|40"}}}} -->
+<div class="wp-block-group" style="margin-top:var(--wp--preset--spacing--40)"><!-- wp:post-featured-image {"align":"full"} /-->
```

Comment:
I think this element should still use a `main` tag. And should it still have the `lock` settings?